src/features/Console.ts

```diff
@@ -220,5 +247,31 @@ export class ConsoleFeature implements IFeature {
         this.languageClient.onRequest(
             ShowInputPromptRequestType,
             (promptDetails) => showInputPrompt(promptDetails, this.languageClient));
+
+        // Set up status bar alerts for when PowerShell is executing a script
+        this.languageClient.onNotification(
+            ExecutionStatusChangedNotificationType,
+            (executionStatusDetails) => {
+                // Dummy promise to satisfy the arguments of `showAnimatedStatusBarMessage`
+                const never = new Promise((resolve, reject) => {
+                    return;
+                });
+
+                switch (executionStatusDetails.executionStatus) {
+                    // If execution has changed to running, make a notification
+                    case ExecutionStatus.Running:
+                        this.executionStatusBar =
+                            AnimatedStatusBar.showAnimatedStatusBarMessage("POWERSHELL EXECUTING", never);
+                        break;
+
+                    // If the execution has stopped, destroy the previous notification
+                    case ExecutionStatus.Completed:
```

Comment:
Does tslint not yell at you for missing the break statement in cases? That's surprising to me.

Reply:
Don't think so. It's set to error on any linting warnings, so it shouldn't have compiled if it did. I'll add it in though -- I recall thinking about it (and wondering why tslint didn't care), but I must have forgotten to put it in.

Reply:
Oh I see, you mean tslint shouldn't allow fallthrough? I would have put `default` but I want to ignore `ExecutionStatus.Pending`. So fallthrough seemed to match my needs best. Do we want to avoid that? I can change if so :)

Reply:
eh. I'm fine with fallthrough but I know it's controversial so I was surprised :)